frontend/src/component/PublicProfile/CustomFormManager.jsx

```diff
@@ -19,19 +19,20 @@ const DisplayTitle = props => {
   )
 }
 
+const DisplaySchemaPropertyLabel = translate()(props => (
+  (props.label && props.label.trim())
+    ? (
+      <span className='DisplaySchemaPropertyString__label'>
+        {props.t('{{label}}: ', { label: props.label })}
+      </span>
+    )
+    : undefined
```

Comment:
React doesn't like "undefined" as component. If you want to return "nothing", always return `null`

Reply:
```suggestion
    : null
```